astropy/cosmology/tests/test_cosmology.py

```diff
@@ -431,25 +431,25 @@ def test_flat_z1():
 def test_zeroing():
     """ Tests if setting params to 0s always respects that"""
     # Make sure Ode = 0 behaves that way
-    cosmo = core.LambdaCDM(H0=70, Om0=0.27, Ode0=0.0)
+    cosmo = flrw.LambdaCDM(H0=70, Om0=0.27, Ode0=0.0)
     assert allclose(cosmo.Ode([0, 1, 2, 3]), [0, 0, 0, 0])
     assert allclose(cosmo.Ode(1), 0)
     # Ogamma0 and Onu
-    cosmo = core.FlatLambdaCDM(H0=70, Om0=0.27, Tcmb0=0.0)
+    cosmo = flrw.FlatLambdaCDM(H0=70, Om0=0.27, Tcmb0=0.0)
     assert allclose(cosmo.Ogamma(1.5), [0, 0, 0, 0])
     assert allclose(cosmo.Ogamma([0, 1, 2, 3]), [0, 0, 0, 0])
     assert allclose(cosmo.Onu(1.5), [0, 0, 0, 0])
     assert allclose(cosmo.Onu([0, 1, 2, 3]), [0, 0, 0, 0])
     # Obaryon
-    cosmo = core.LambdaCDM(H0=70, Om0=0.27, Ode0=0.73, Ob0=0.0)
+    cosmo = flrw.LambdaCDM(H0=70, Om0=0.27, Ode0=0.73, Ob0=0.0)
     assert allclose(cosmo.Ob([0, 1, 2, 3]), [0, 0, 0, 0])
 
 
 # This class is to test whether the routines work correctly
 # if one only overloads w(z)
-class test_cos_sub(core.FLRW):
+class test_cos_sub(flrw.FLRW):
     def __init__(self):
-        core.FLRW.__init__(self, 70.0, 0.27, 0.73, Tcmb0=0.0,
```

Comment:
Maybe change to `super().__init__(70.0, ...)` while you are at it?

Reply:
👍   ✅